### DISCUSSION TITLE
Proper way to ignore presence events that are only time stamp changes

### DISCUSSION THREAD

**Discussion by JMTK** [discordjs/discord.js#5336] [Q&A] [answered]
Due to the nature of rich presences, you can receive many presence update events, only to realize that it's just the elapsed game time for a match or game being updated constantly. I would like to be able to generally filter out these types of updates early on in my presence handler and only handle more substantial updates. 

Would the best way to go about this be to recursively compare old/new presence for all properties and see if only the `timestamps` property changed?

Looking for best practice ideas!

**Comment by DaStormer** [comment]
If you don't have a need for `timestamps`, maybe delete that property and then compare the two presences?

**Comment by vladfrangu** [comment, marked as answer]
Since you get the old and new presence, you can always check if the names/ids/whatevers are the same, and check if timestamps changed - then you can safely ignore that event! Personally, that sounds like the best approach.
